ckanext/restricted/action.py

```diff
@@ -75,7 +75,8 @@ def body_from_user_dict(user_dict):
         mail_recipient(name, email, subject, body)
 
     except MailerException as mailer_exception:
-        log.error('Cannot send mail after registration')
+        log.error('Cannot send mail after registration for user {}'.format(user_dict.get('name')))
+        log.error('Email address that is causing this issue: {}'.format(user_dict.get('email')))
```

Comment:
Better replace with a single line log:
```
log.error(f'Cannot send mail after registration for user {user_dict.get('name')} and email {user_dict.get('email')', exc_info=True)
```

Reply:
Thanks, I've changed it to what you've proposed. BTW, you can submit code changes as a suggestion in GH, they can be then directly applied from Pull Request: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request